File: src/pizza/blackolivepineapple/sentinel/ast/VariableDeclStatementNode.java
package pizza.blackolivepineapple.sentinel.ast;

public class VariableDeclStatementNode extends StatementNode{
    String typeName, varName;
    ExpressionNode expressionNode;
    public VariableDeclStatementNode(String typeName, String varName, ExpressionNode expressionNode){
        this.typeName = typeName;
        this.varName = varName;
        this.expressionNode = expressionNode;
    }
}

File: ast/VariableDeclStatementNode.cpp
#include "VariableDeclStatementNode.h"

VariableDeclStatementNode::VariableDeclStatementNode(std::string typeName, std::string varName, ExpressionNode expressionNode) {
    this->typeName = typeName;
    this->varName = varName;
    this->expressionNode = expressionNode;
}